correlation_test.go

```diff
@@ -8,12 +8,13 @@ import (
 	"github.com/montanaflynn/stats"
 )
 
-func ExampleCorrelation() {
+func ExampleCorrelation(l *testing.T) {
 	s1 := []float64{1, 2, 3, 4, 5}
 	s2 := []float64{1, 2, 3, 5, 6}
 	a, _ := stats.Correlation(s1, s2)
-	fmt.Println(a)
-	// Output: 0.9912407071619302
+	if !veryclose(a, 0.9912407071619302) {
```

Comment:
I think we could use `stats.Round` here with the same result and added benefit of showing off another exported function instead of a private function that cannot be copy / pasted as one would expect from an example.